modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -110,3 +110,79 @@ def func_to_apply(partition, row_internal_indices, col_internal_indices, item):
     md_df._query_compiler._modin_frame = new_modin_frame
 
     df_equals(md_df, pd_df)
+
+
+small_dfs = [
+    pd.DataFrame(
+        [[i + j for j in range(0, 1000)]],
+        columns=[f"col{j}" for j in range(1, 1001)],
+        index=pd.Index([i - 1]),
+    )
+    for i in range(1, 100001, 1000)
+]
+large_df = pd.DataFrame(
```

Comment:
If you move the concats out of the params, I think you should move these definitions inside the test body.